app/assets/javascripts/components/settings/containers/special_user_container.jsx

```diff
@@ -0,0 +1,4 @@
+import { connect } from 'react-redux';
+import SpecialUser from '../views/special_user';
+
+export default connect()(SpecialUser);
```

Comment:
This is unnecessary. `connect` isn't doing anything here; you may as well just use SpecialUser directly in SettingsHandler.

Reply:
The file was intended as a placeholder, we will need it to add a new action. I think I should make a complete implementation as you suggested.
this implementation is not obvious.